doc/cookbook/using-free-client/UsingFreeClient.lhs

```diff
@@ -0,0 +1,173 @@
+# Using Free Client (for tests)
+
+Someone asked on IRC about getting `Request` & `Response` of what
```

Comment:
I'd suggest:

Someone asked on IRC how one could access the intermediate `Request`s (resp. `Response`s) produced (resp. received) by client functions derived using servant-client.